run_tests.js

```diff
@@ -29,8 +30,10 @@ function runTests() {
       svgAnchor.innerHTML = testData[name].svg;
 
       var preserveFill = testCases[i].preserveFill || false;
+      
+      var fixViewbox = testCases[i].fixViewbox || false;
 
-      var output = ProcessSvg(svgAnchor.querySelector('svg'), 1, 1, 0, 0, preserveFill);
+      var output = ProcessSvg(svgAnchor.querySelector('svg'), 1, 1, 0, 0, preserveFill, fixViewbox);
```

Comment:
nit: maybe rather than `fixViewbox` everywhere it should be `normalizeCoordinates`? You're changing all the coordinates in the paths, not just the viewbox.